www/docs/constructs/Service.about.md

```diff
@@ -321,7 +321,7 @@ new Service(stack, "MyService", {
 
 ### Configuring Service Container
 
-Here's an example of configuring the Fargate container health check.
+Here's an example of configuring the Fargate container health check. Make sure the `curl` command exists inside the container.
```

Comment:
Spent more time than I care to admit working out why my healthcheck was failing 😅